### PR TITLE
Create shortcode for embedded c-span videos

### PR DESCRIPTION
**Pull request by nick-mon1**
### Summary

Adds a new `cspan-iframe` shortcode for embedding iframes from cspan.

How to use:
```
{{< cspan-iframe src="https://www.c-span.org/video/standalone/?512480-1/confirmation-hearing-gsa-administration-intelligence-agencies-nominees" title="Confirmation Hearing for GSA Administration and Intelligence Agencies Nominees" >}}
```

[Preview Page](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-video-embed-shortcode/2023/02/18/we-asked-our-design-team-what-did-you-learn-in-2022-and-what-are-you-looking-forward-to-in-2023/)

Screen Shot 2023-02-21 at 4 10 57 PM

### iFrame Styles

We are not able to override the `iframe` styles, hence the blue border that is slightly clipped on the right side.

### Checklist

- [x] `src` and `title` attributes
- [x] responsive
- [x] tested on browsers — chrome, firefox, safari
- [x] `sandbox` attribute on iframe for security settings, see MDN docs more [details](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox)
  - [x] included `allow-same-origin allow-scripts`
- [ ] update shortcodes page in wiki
- [x] remove test page before merging @ToniBonittoGSA

### CODE REVIEW

File: themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    height: 100%;
+    width: 100%;
+  }
+
+  // iframe classes to override to create perfectly fit video player
+  body.video.standalone section {
+    background-color: none;
+    margin-top: 0;
+    padding: 0; 
+  }
```
Comment (mejiaj) [inline themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss:13]: Let's remove this if it doesn't work.

File: themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.cspan-embed {
+  iframe {
+    aspect-ratio: 4/3;
+    height: 100%;
+    width: 100%;
+  }
```
Comment (mejiaj) [inline themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss:6]: You could remove these styles by using USWDS's aspect class `add-aspect-4x3`.

File: themes/digital.gov/layouts/shortcodes/cspan-iframe.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{{/*
+  Pass in the src url and the title attributes
+  {{< cspan-iframe src="https://www.c-span.org/video/standalone/?512480-1/confirmation-hearing-gsa-administration-intelligence-agencies-nominees" title="Confirmation Hearing for GSA Administration and Intelligence Agencies Nominees" >}}
+*/}}
+  
+{{- $src := .Get "src" -}}
+{{- $title := .Get "title" -}}
+
+<section class="cspan-embed">
```
Comment (mejiaj) [inline themes/digital.gov/layouts/shortcodes/cspan-iframe.html:9]: Consider adding aspect class and removing custom styles. I'd still keep the cspan-embed class for future use. So something like: `class="cspan-embed add-aspect-4x3"`.

**Review by mejiaj — CHANGES_REQUESTED** [review]
@nick-mon1 minor suggestions, the embed works as expected. Unfortunately we can't override the border/padding that's being added in the iframe content.

File: themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss (hunk 1 above)
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss:13]: Removed

File: themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss (hunk 2 above)
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss:6]: Replaced with utility class

Comment (mejiaj) [reply themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss]: Thank you! Resolved in [6944183](https://github.com/GSA/digitalgov.gov/pull/5827/commits/6944183eeac8a0d3ca786fe71927a51e4f0866fe).

File: themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss (hunk 1 above)
Comment (mejiaj) [reply themes/digital.gov/src/scss/new/shortcodes/_cspan-iframe.scss]: Removed in [6944183](https://github.com/GSA/digitalgov.gov/pull/5827/commits/6944183eeac8a0d3ca786fe71927a51e4f0866fe).

**Review by mejiaj — DISMISSED** [review]
Nice work, I can view and play the cspan shortcode.

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**